services/subscriber.py

```diff
@@ -20,11 +20,11 @@ def get_subscribers(self, targetings, hours_whitelist, volume):
         if timezones:
             targetings.append({
                 "field": "timezone",
-                "operator": "AND",
+                "operator": "IN",
                 "value": timezones
             })
         try:
-            s = Search(using=es, index="subscribers")
+            s = Search(using=es, index="users")
             s.extra(size=volume, boost_mode='replace')
```

Comment:
I think, s.extra(size=limit) works fine. I used it in my other Django project. Do you think, I have to change it?